docker-compose.yml

```
@@ -15,10 +37,26 @@ services:
+      POSTGRES_HOST: "${POSTGRES_HOST}"
+      POSTGRES_PORT: "${POSTGRES_PORT}"
+      POSTGRES_USER: "${POSTGRES_USER}"
+      POSTGRES_PASSWORD: "${POSTGRES_PASSWORD}"
+      USER_DB_NAME: "${USER_DB_NAME}"
     volumes:
       - ${HOME}/videos:/videos
+    depends_on:
+     postgres:
+       condition: service_healthy
+    healthcheck:
+     test: ps aux | grep python
```

yamllint (1.35.1)

[warning] wrong indentation: expected 6 but found 5

(indentation)